app/src/main/java/com/android/brewr/ui/explore/Map.kt

```diff
@@ -100,10 +124,33 @@ private fun getMarkerIcon(shopName: String): BitmapDescriptor {
   }
 
   return when {
-    shopName.contains("Starbucks", ignoreCase = true) ->
+    coffee.coffeeShopName.contains("Starbucks", ignoreCase = true) ->
         loadAndResizeIcon(R.drawable.starbucks_icon)
-    shopName.contains("McDonald's", ignoreCase = true) ->
+    coffee.coffeeShopName.contains("McDonald's", ignoreCase = true) ->
         loadAndResizeIcon(R.drawable.mcdonald_icon)
+    isJourney(coffee, listJourneysViewModel) -> loadAndResizeIcon(R.drawable.journeys_icon)
     else -> loadAndResizeIcon(R.drawable.default_coffee_icon) // Default icon
   }
 }
+
+/**
+ * Checks if the given coffee location is part of any journey in the listJourneysViewModel.
+ *
+ * @param coffee The coffee object containing the location to check.
+ * @param listJourneysViewModel The ViewModel containing the list of journeys.
+ * @return True if the coffee location is part of any journey, false otherwise.
+ */
+@Composable
```

Comment:
The way of using epsilon seems a very good idea. Have you tried with other small numbers as epsilon (0.001, 0.1 etc) ? For you, the best epsilon was 0.01 ? Anyways it's good to set a "margin of error" ! (I don't know if it's the correct expression though)

Reply:
Yes, I tried with few examples, 0.01 is the good balance between permissive and strict